pipeline/column.csv

```diff
@@ -1 +1,2 @@
 dataset,resource,column,field
+flood-risk-zone,,objectid,reference
```

Comment:
Might not need solving now but we should reach out to the data provider to check if the object ID is a UID that doesn't change between releases